### ISSUE TITLE
1.1.1 Release install 0.0.4 version of flake8-django checker module

### ISSUE THREAD

**Issue by playpauseandstop** [rocioar/flake8-django#88]
**TLDR:** `pip install flake8-django==1.1.1` install `checker` module from `flake8-django==0.0.4`

---

OS Details:

- macOS Catalina (10.15.7)
- Python 3.9.4 (installed via pyenv)

Steps to reproduce,

1. Create new virtualenv with, `python3 -m venv flake8-django-venv`
2. Install `flake8==3.9.4 flake8-django==1.1.1` into `flake8-django-venv`
3. Run `flake8 --help | grep --color flake8-django`
4. Expect to have `1.1.1` as installed version of `flake8-django`, but have `0.0.4` instead

```shell
❯ python3 --version
Python 3.9.4

❯ python3 -m venv flake8-django-venv

❯ flake8-django-venv/bin/pip install flake8==3.9.1 flake8-django==1.1.1
Collecting flake8==3.9.1
  Using cached flake8-3.9.1-py2.py3-none-any.whl (73 kB)
Collecting flake8-django==1.1.1
  Using cached flake8_django-1.1.1-py3-none-any.whl (23 kB)
Collecting pyflakes<2.4.0,>=2.3.0
  Using cached pyflakes-2.3.1-py2.py3-none-any.whl (68 kB)
Collecting pycodestyle<2.8.0,>=2.7.0
  Using cached pycodestyle-2.7.0-py2.py3-none-any.whl (41 kB)
Collecting mccabe<0.7.0,>=0.6.0
  Using cached mccabe-0.6.1-py2.py3-none-any.whl (8.6 kB)
Installing collected packages: pyflakes, pycodestyle, mccabe, flake8, flake8-django
Successfully installed flake8-3.9.1 flake8-django-1.1.1 mccabe-0.6.1 pycodestyle-2.7.0 pyflakes-2.3.1
WARNING: You are using pip version 20.2.3; however, version 21.1.1 is available.
You should consider upgrading via the '/Users/playpauseandstop/Projects/flake8-django-venv/bin/python3 -m pip install --upgrade pip' command.

❯ flake8-django-venv/bin/flake8 --help | grep --color flake8-django
Installed plugins: flake8-django: 0.0.4, mccabe: 0.6.1, pycodestyle: 2.7.0,
```

Other way to reproduce, is to download `whl` file [directly from PyPI](https://pypi.org/project/flake8-django/#files), unarchive it and check content of `checker.py` file,

```
❯ curl -O https://files.pythonhosted.org/packages/00/a1/2457403ec15c534b6e1dea562d63e6017cf6bb39b8b1e7f850b0a1d3ef7c/flake8_django-1.1.1-py3-none-any.whl
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 23505  100 23505    0     0   145k      0 --:--:-- --:--:-- --:--:--  145k

❯ unzip -x flake8_django-1.1.1-py3-none-any.whl
Archive:  flake8_django-1.1.1-py3-none-any.whl
  inflating: flake8_django/VERSION   
  inflating: flake8_django/__init__.py  
  inflating: flake8_django/checker.py  
  inflating: flake8_django/checkers/__init__.py  
  inflating: flake8_django/checkers/base_model_checker.py  
  inflating: flake8_django/checkers/checker.py  
  inflating: flake8_django/checkers/issue.py  
  inflating: flake8_django/checkers/model_content_order.py  
  inflating: flake8_django/checkers/model_dunder_str.py  
  inflating: flake8_django/checkers/model_fields.py  
  inflating: flake8_django/checkers/model_form.py  
  inflating: flake8_django/checkers/model_meta.py  
  inflating: flake8_django/checkers/render.py  
  inflating: flake8_django/checkers/urls.py  
  inflating: flake8_django-1.1.1.dist-info/LICENSE  
  inflating: flake8_django-1.1.1.dist-info/METADATA  
  inflating: flake8_django-1.1.1.dist-info/WHEEL  
  inflating: flake8_django-1.1.1.dist-info/entry_points.txt  
  inflating: flake8_django-1.1.1.dist-info/top_level.txt  
  inflating: flake8_django-1.1.1.dist-info/RECORD  

❯ cat flake8_django/VERSION 
1.1.1

❯ grep --color __version__ flake8_django/checker.py 
__version__ = '0.0.4'
    version = __version__
```

---

With that in mind, it would be really nice, if `1.1.2` version of `flake8-django` will be released with proper content of `checker` module.

Thanks and please let me know if you need any additional details.
